src/App.js

```diff
@@ -1,19 +1,62 @@
-import React from 'react';
-import './App.css';
+import React, { useEffect, useState } from 'react';
+import ChatLog from './components/ChatLog';
 import chatMessages from './data/messages.json';
+import './App.css';
 
 const App = () => {
+
+  const [chatEntryData, setChatEntryData] = useState([
+    {
+    id: 0,
+    sender: '',
+    body: '',
+    timeStamp: '',
+    liked: false
+    }
+  ]);
+
+  useEffect(() => {
+    setChatEntryData(chatMessages);
+  }, []);
```

Comment:
Instead of using useEffect here to set the values for chatEntryData on page load, you can omit this completely and directly set the state on line 8. That would look like:
```
  const [chatEntryData, setChatEntryData] = useState(chatMessages);